File: LeetCode/2643. Row With Maximum Ones/Solution.py
class Solution:
    def rowAndMaximumOnes(self, mat: List[List[int]]) -> List[int]:
        maxOnesRowIndex = 0
        maxOnes = 0

        for i, row in enumerate(mat):
            currentOnes = row.count(1)

            if currentOnes > maxOnes:
                maxOnes = currentOnes
                maxOnesRowIndex = i

        return [maxOnesRowIndex, maxOnes]

File: LeetCode/2643. Row With Maximum Ones/Solution.cpp
class Solution {
public:
    vector<int> rowAndMaximumOnes(vector<vector<int>>& mat) {
        int maxOnesRowIndex = 0;
        int maxOnes = 0;

        for (int i = 0; i < mat.size(); i++) {
            int currentOnes = count(mat[i].begin(), mat[i].end(), 1);

            if (currentOnes > maxOnes) {
                maxOnes = currentOnes;
                maxOnesRowIndex = i;
            }
        }
        
        return vector<int> {maxOnesRowIndex, maxOnes};
    }
};
